stdlib/expand_module_aliases.awk

```diff
@@ -22,8 +22,9 @@ NR == 1 { printf ("# 1 \"%s\"\n", FILENAME) }
   else if (state==1)
     state=2;
   else if ($1 == "module")
-    printf ("\n(** @canonical %s *)\nmodule %s = Stdlib__%s%s\n",
-            $2, $2, tolower(substr($4,1,1)), substr($4,2));
+#   printf("\n(** @canonical %s *)", $2);
```

Comment:
Why was this commented BTW? I believe it matters for odoc, for instance if someone tries to generate the HTML doc for all installed packages, it might look better with this tag

Reply:
In this case, I fixed the awk script by not printing this line only if the variable ocamldoc is set to true. Like this, ocamldoc does not see an unknown custom tag, and odoc can generate better documentation.

Reply:
Cool, thanks